src/sidebar/templates/timestamp.html

```diff
@@ -0,0 +1,12 @@
+<div ng-switch="vm.href">
+  <span ng-switch-when="">
```

Comment:
The `class` attribute is missing here so the font size and color does not match between linked vs. unlinked timestamps.

Also the title attribute is missing so the full annotation creation time won't be shown on hover.

Reply:
Nice catch, cheers